front-end/src/app/shared/job-card/job-card.component.scss

```diff
@@ -0,0 +1,120 @@
+*, :after, :before {
+  box-sizing: border-box;
+}
+
+.listing {
+  text-align: center;
+  background: var(--back-color);
+  border: 1px solid #e7e7e7;
+  border-radius: 3px;
+  overflow: hidden;
+  margin: 10px 15px 30px;
+  flex: 1 1 300px;
+  min-width: 200px;
+}
+
+.cover {
+  border-radius: 3px 3px 0 0;
+  position: relative;
+  height: 130px;
+  padding-top: 56.25%;
+  overflow: hidden;
+}
+
+.cover-wrapper {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0
+}
+
+.cover img {
+  min-height: 100%;
+  min-width: 100%;
+  max-height: 100%;
+  max-width: 100%;
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+  overflow-clip-margin: content-box;
+  overflow: clip;
+  border-radius: 3px 3px 0 0;
+  border-style: none;
+}
+
+img, svg {
+  vertical-align: middle;
+}
+
+.info {
+  margin-top: 10px;
+  padding: 0 20px;
+  justify-content: space-around
+}
+
+.info h5 {
+  color: #9bb2bf;
+  font-size: 13px;
+  margin-bottom: 3px;
+}
+
+.info h3 {
+  font-size: 16px;
+  margin-bottom: 5px;
+  color: white;
+  align-items: center;
+  justify-content: center;
+}
+
+.bottom-location {
+  border-top: 1px solid #e7e7e7;
+  padding: 6px 12px;
+  color: #9bb2bf;
+  font-size: 12px;
+  max-height: 30px;
+}
+
+.details {
+  border-top: 1px solid #e7e7e7;
+  display: flex;
+  height: 100px;
+  width: 230px;
+}
+
+.salary {
+  border-right: 1px solid #e7e7e7;
+  width: 115px;
```

Comment:
try to size the flexbox items with flex-basis or try other methods [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)